### PR TITLE
Ifteda About Page 1st Draft

### PR DESCRIPTION
**Pull request by ifteda**
TODOs listed at top of About.js

### CODE REVIEW

File: src/styling/about.css
Comment (feliciazhang) [inline src/styling/about.css]: I don't think you need this bit since the fonts already imported

File: src/components/AboutPage.js
Comment (feliciazhang) [inline src/components/AboutPage.js]: if you get rid of class=p here, the css rule can just look like
```
p { ... }
```
instead of .p

File: src/components/AboutPage.js
Comment (feliciazhang) [inline src/components/AboutPage.js]: in react everything should actually be `className=` and not class sorry this is my bad i didnt change them

File: src/components/AboutPage.js
Comment (feliciazhang) [inline src/components/AboutPage.js]: i'm just being nitpicky now lol you don't have to change this but usually it's better to use css and adjust margins/padding rather than adding linebreaks/br tags

**Review by feliciazhang — APPROVED** [review]

File: src/components/AboutPage.js
Comment (ifteda) [reply src/components/AboutPage.js]: lol keep the suggestions coming. i'll adjust

File: src/styling/about.css
Comment (liu-caro) [inline src/styling/about.css]: I made a var for white as well if you want to change it to that to be consistent!

File: src/components/AboutPage.js
Comment (liu-caro) [inline src/components/AboutPage.js:14]: could you add some margin at the top so it looks even (currently being eaten up by navbar)

File: src/components/AboutPage.js
Comment (liu-caro) [inline src/components/AboutPage.js:110]: could we add some margin at the bottom so that the page doesn't end abruptly?

File: src/components/AboutPage.js
Comment (liu-caro) [inline src/components/AboutPage.js]: This would look better with a mail icon: email. 
And then a single row of the other socials can just be the logos as links.
(Total: 2 rows)

File: src/styling/about.css
Comment (feliciazhang) [inline src/styling/about.css]: is there a px margin that would work? %s are a bit iffy bc it'll look different on different screens

File: src/components/AboutPage.js
Comment (feliciazhang) [inline src/components/AboutPage.js]: all style sheets should be imported into one place in index.css
https://github.com/nupur-n/build4blm/blob/dev/src/index.css#L1

Comment (ifteda) [reply src/components/AboutPage.js]: i think i need to merge dev into this branch. my index.css seems outdated

File: src/components/AboutPage.js
Comment (ifteda) [reply src/components/AboutPage.js:14]: ya was fiddling around with spacing but haven't pushed yet

File: src/components/AboutPage.js
Comment (ifteda) [reply src/components/AboutPage.js]: waiting on decision for icon pack to use

File: src/components/AboutPage.js
Comment (ifteda) [reply src/components/AboutPage.js]: done. question: in about.css, i specify some things for elements (i.e. hr, p, ul) rather than classes. should I modify to use classes instead to avoid conflicts with other pages' styling?

File: src/components/AboutPage.js
Comment (nupur-n) [inline src/components/AboutPage.js:51]: this is nitpicky but since the background is black you can't see the bullet so it just looks a little awkward spaced inline w/ only one partner - could we maybe add something like "contact us if you want to get involved or join our slack" kind of language for people who want to be community partners? And then unindent HBP for now or change the bullet color?

**Review by nupur-n — APPROVED** [review]
overall looks good to me! I added some minor feedback but think it'd good to merge if you want to make those fixes for another PR.

File: src/components/AboutPage.js
Comment (ifteda) [reply src/components/AboutPage.js:51]: O I removed the bullets, but I'll un-indent to better match the wireframe.
When we ask them to reach out, should we link our email again, or would that be redundant since it's already mentioned later on the same page?

**Pull request merged**
